c7n/mu.py

```diff
@@ -43,6 +43,7 @@
 
 
 log = logging.getLogger('custodian.lambda')
+DEFAULT_RUNTIME = {2: 'python2.7', 3: 'python3.6'}[sys.version_info[0]]
```

Comment:
This construction allows for running `custodian` under, e.g., 3.5 locally and still pushing a 3.6 lambda.

Reply:
yup, sounds good, sorry missed that index into the dict first time through, deleted original comment.